### ISSUE TITLE
Playback crashes on plex

### ISSUE THREAD

**Issue by xackery** [crunchy-labs/crunchy-cli#230]
**Describe the bug**
When playing back media, a bug occurs where the subtitle text freezes while the video continues playing, and eventually the player crashes with a message attached on screenshot. It seems to imply the mkv is invalid.

**To Reproduce**
Steps / command to reproduce the behavior:
```
$ crunchy-cli archive --skip-existing -a ja-JP -s en-US -r 480x360 -o "s{season_number}e{episode_number}.mkv" https://www.crunchyroll.com/series/GMEHME55K/the-worlds-finest-assassin-gets-reincarnated-in-another-world-as-an-aristocrat[S1E8]
```

This occurs I think at random. E.g. in this case, it crashed at 12:48, 10 minutes left. I don't know if the location changes, when I hit resume after the crash it seems to play fine. But then crashes again at a different location. The frequency is about once per episode, some times more, some times less.

I changed my burn in behavior to "Only Image formats" as without it, as you know, it caused the font to be incorrect. I may go download the needed fonts of the previous issue and see if this issue exists, due to the symptoms I'm guessing it's a malformed subtitle or something?

**Expected behavior**
Not to crash.

**Screenshots**
Error is this:
<img width="483" alt="image" src="https://github.com/crunchy-labs/crunchy-cli/assets/845670/82a441dc-2368-4929-bece-408c5418ccc1">

**Client (please complete the following information):**
 - OS: ubuntu
 - Version: crunchy-cli 3.0.0-dev.14 (f4682e0 2023-06-19)

**Additional context**
I recently fixed the subtitle issue earlier.

**Comment by BlastBolt5** [comment]
Just thought i'd throw my input in here as well.
My plex does the same thing with only image formats on, on auto everything works fine with no crashes tho

**Comment by xackery** [comment]
A follow up, this seems to occur on darwin with various browsers, but on the plex app the issue does not occur. Playback setting does not seem to fix it fully. I'll keep investigating..

**Comment by xackery** [comment]
After updating to the most recent 3.0.0 release, I haven't seen it yet however..

**Comment by BlastBolt5** [comment]
I've seen it occur on brave for me mainly, luckily it seems like the original issue of formatting of subs on plex with subtitle settings set to auto has been fixed in the latest update of plex, but the fact of why it's crashing could either be another plex bug or bad formatting according to spec on the subtitles, which idk if crunchy-cli actually messes with subtitles at all of if they are just left as downloaded. since you haven't seen the problem yet with 3.0.0 i'll update mine to that and see if it fixes the issue, it possible it has something to do with the fix from #66 but idk if that's something that could break plex

**Comment by xackery** [comment]
You bring up a good point, there's a chance plex recently fixed it and I just so happened to update latest crunchy-cli as well.. and they may not correlate.

**Comment by ByteDream** [comment]
How is the state of this issue? Did a plex update fix it?

**Comment by BlastBolt5** [comment]
the squashed burned subs have been fixed in the latest plex server beta, but idk about the crashing, it's random so hard to test

**Comment by xackery** [comment]
I experienced it again with the browser version of plex on OSX.
I am now just using the app form and that removes it. I have a strong suspicion it's purely plex's side and not much can be done on your part, minus maybe considering not encoding a windows exclusive font.. I'll go ahead and close this though, not because it's confirmed done, but because I found a personal fix and blastbolt's suggestion may mean the other variant may be fixed too.
